pages/community/dei-mission-statement.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+permalink: /dei-mission/
+---
+
+The US-RSE Association is committed to providing an [inclusive environment with
+equitable treatment]({{site.url}}/code-of-conduct/) for all and to promoting
+and encouraging diversity throughout the RSE community in the US. At US-RSE,
+integrating DEI practices into our education programs, governance structure,
+and culture is at the forefront of our mission to ensure a welcoming,
+nurturing, and robustly inclusive community. We believe that the amplification
+of diverse perspectives is essential for driving innovation, promoting
+creativity, and encouraging engagement for the success of RSEs. We welcome and
+respect individuals of all dimensions of diversity, including but not limited
```

Comment:
I didn't come to the final meeting, and maybe it's okay because it says "including but not limited to" but we don't make any reference to any kind of neurodiversity. At least that one feels important to me.

Reply:
This was one of the points we spent a bit more time on. We had a few additional "dimensions" from the "public" comment period. At the time we decided to cut a couple as we didn't believe a long running list or potentially ever changing list was appropriate. This is why we decided to add the "including but not limited to" and not include everything.

I'm not sure that's the best answer, but from my recollection that was the reasoning. What do others think and feel about this specific sentence?

Reply:
I think if we are hitting the major categories, we should include something (also general) like neurocognitive differences. https://trainingindustry.com/articles/diversity-equity-and-inclusion/neurodiversity-in-the-age-of-di-how-do-training-professionals-make-an-impact/. That covers a whole lot of stuffs.

Reply:
I'm not sure - on the one hand, we can't include everything, but on the other, we do want to include the major groups. I'm honestly not sure which category this fits it